### ISSUE TITLE
Bugfix: Harvest Moon 64 FPS

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#173]
```
What steps will reproduce the problem?
1. This will work during any stage of gameplay
2.
3.

What is the expected output? What do you see instead?
Harvest Moon 64 is usually sluggish sitting at around 50-60 FPS but if you
disable audio in Wii64's settings the FPS jumps to 98 and stays there
making it almost identical to the console version. 

What version of the product are you using? On which system and loader?
Wii64 / Wii / Homebrew Channel

Please provide any additional information below.
This might work for other games that are slow during normal game play. 

```

Original issue reported on code.google.com by `Irenicus...@gmail.com` on 30 Nov 2009 at 7:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `emuki...@gmail.com` on 1 Feb 2010 at 11:34
- Changed title: **Bugfix: Harvest Moon 64 FPS**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in > v1.1
```

Original comment by `emuki...@gmail.com` on 9 Oct 2012 at 12:08
- Changed state: **Fixed**
